libs/design-system/cards/gallery-card/src/lib/gallery-card.component.scss

```diff
@@ -0,0 +1,67 @@
+@use '../../../../styles/utils';
+@use '../../../../styles/vars';
+
+:host {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  flex-grow: 1;
+  width: 100%;
+  min-width: 17rem;
+  max-width: 28rem;
+
+  .header {
+    .image {
+      width: 100%;
+      max-height: 12.5rem;
+      object-fit: cover;
+      aspect-ratio: 16 / 9;
+    }
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    padding-top: 1rem;
+    min-height: 6.875rem;
+
+    .date {
+      @include utils.use-font(label, medium);
+      color: vars.$on-secondary-fixed;
+    }
+
+    .tagline,
+    .tagline-link {
+      @include utils.use-font(title, medium);
+      color: vars.$tertiary;
+      text-decoration: underline;
+      display: -webkit-box;
```

Comment:
Do not use vendor specific layouts. Use `block`, `flex`, etc. to achieve the correct layout